cpp/tests/stream_compaction/apply_boolean_mask_tests.cpp

```diff
@@ -143,6 +143,30 @@ TEST_F(ApplyBooleanMask, withoutNullString)
   CUDF_TEST_EXPECT_TABLES_EQUAL(expect_cudf_table_view, tableView);
 }
 
+TEST_F(ApplyBooleanMask, FixedPointColumnTest)
```

Comment:
I would add a test with greater than 1000 elements as well.

Reply:
+1

Reply:
The test with large decimal column has been added.

Reply:
@sperlingxx :+1: Note: you can request a "re-review" once you have addressed all requested changes by a reviewer by click the two arrows in a circle.
![image](https://user-images.githubusercontent.com/36027403/99881923-c3aa6d80-2bea-11eb-9f18-2bc3a4546100.png)